raw_description/urdf/arm_camera/arm_camera.urdf.xacro

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:include filename="$(find cob_description)/urdf/common.xacro"/>
+  <xacro:include filename="$(find raw_description)/urdf/arm_camera/arm_camera.gazebo.xacro" />
```

Comment:
we have sensors in `cob_description/urdf/sensors`...you could put it there
I would also remove the prefix `arm_` and keep the sensor "mount-position"-independent
I would even rather use something like manufacturer_model.urdf.xacro as a name - if possible (see the other sensors in `cob_description`)